canonical-web-and-design/maas-ui
# feat(ui): remove legacy domain files + add redirects

## Done

- Changed DNS nav link to React version
- Removed legacy domain files
- Added redirects for legacy domain URLs
- Added redirects for legacy zone URLs
- Updated domains cypress tests

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Run the full UI and navigate to /MAAS/l/domains
- Check that you are redirected to the React domain list
- Navigate to /MAAS/l/domain/0 and check that you are redirected to the React domain details page for domain 0
- Click on DNS in the nav and check you are taken to the React domain list

## Fixes

Fixes canonical-web-and-design/app-squad#125
